docs/advanced/node-local-storage.md

```diff
@@ -0,0 +1,359 @@
+        image: k8s.gcr.io/busybox
+        command:
+        - "/bin/sh"
+        args:
+        - "-c"
+        - "sleep 100000"
+        volumeMounts:
+        - name: local-vol
+          mountPath: /usr/test-pod
+  volumeClaimTemplates:
+  - metadata:
+      name: local-vol
+    spec:
+      accessModes: [ "ReadWriteOnce" ]
+      storageClassName: "local-storage"
+      resources:
+        requests:
+          storage: 1Gi
+```
+
+The PVC should be listed and the PV bound:
+
+```sh
+$ kubectl get pods
+NAME           READY   STATUS    RESTARTS   AGE
+local-test-0   1/1     Running   0          54s
+local-test-1   1/1     Running   0          48s
+
+$ kubectl get pvc
+NAME                     STATUS   VOLUME        CAPACITY   ACCESS MODES   STORAGECLASS    AGE
+local-vol-local-test-0   Bound    worker-1-pv   10Gi       RWO            local-storage   7s
+local-vol-local-test-1   Bound    worker-0-pv   10Gi       RWO            local-storage   1s
+
+$ kubectl get pv
+NAME          CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                            STORAGECLASS    REASON   AGE
+worker-0-pv   10Gi       RWO            Retain           Bound    default/local-vol-local-test-1   local-storage            97m
+worker-1-pv   10Gi       RWO            Retain           Bound    default/local-vol-local-test-0   local-storage            97m
+```
+
+You can also connect to the pod and write to the volume which is mounted on
+`/usr/test-pod`, you will see in the host on `/mnt`.
+
+If you delete a pod, as it is part of this Stateful Set, it will be recreated and
+continue to use the volume.
+
+**What if you want to delete this application and use the PVs for a different
+application?**
+
+Basically, as the retain policy is set to `Retain` (just to be safe), you need
+to do some manual operations documented [here][1]. Note that when using the `Delete`
+policy this is different, so you may want to choose a policy that better fits
+your use case.
+
+[1]: https://kubernetes.io/docs/concepts/storage/persistent-volumes/#retain
+
+### Automatic Local Persistent Volume Creation and Handling
+
+The example manually created the `Kind: PersistentVolume` for each node. Also,
+the PV will continue to exist even if the PVC is deleted and, therefore, the
+lifecycle of the PV needs to be handled manually.
+
+Those limitation don't scale well with the number of nodes and applications,
+for that reason an external provisioner that will automatically create those
+objects when nodes are available and manage the lifecycle of the volume was
+created by sig-storage:
+	https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/
+
+### Limitations
+
+There are some limitations you might want to have in mind:
+
+1. Lokomotive Kubernetes currently only exposes `/mnt/` to the kubelet, so only that
+directory can be used for node local storage. You can **manually** mount different
+devices on `/mnt/sda` or paths inside `/mnt` at your convenience.
```

Comment:
This was not the case before self-hosted kubelet. Not sure if it is now

Reply:
```suggestion
Lokomotive Kubernetes currently only exposes `/mnt/` to the kubelet, so only that
directory can be used for node local storage.
```

Reply:
If' say remove the "You can **manually** mount different devices on `/mnt/sda` or paths inside `/mnt` at your convenience" but I'm failing to create a github suggestion for this, not sure why :man_facepalming: